lambdas/answer/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandler.java

```diff
@@ -71,22 +79,27 @@ public APIGatewayProxyResponseEvent handleRequest(
         try {
             processAnswerResponse(input);
         } catch (JsonProcessingException jsonProcessingException) {
-            LOGGER.error("Failed to parse object using ObjectMapper");
+            eventProbe.log(INFO, jsonProcessingException).counterMetric(POST_ANSWER, 0d);
             response.withStatusCode(HttpStatus.SC_INTERNAL_SERVER_ERROR);
-            response.withBody("{ " + ERROR + ":\"" + jsonProcessingException.getMessage() + "\" }");
+            response.withBody("{ " + ERROR + ":\"Failed to parse object using ObjectMapper.\" }");
         } catch (NullPointerException npe) {
-            LOGGER.error("Error finding the requested resource");
+            eventProbe.log(INFO, npe).counterMetric(POST_ANSWER, 0d);
             response.withStatusCode(HttpStatus.SC_BAD_REQUEST);
-            response.withBody(npe.getMessage());
+            response.withBody("{ " + ERROR + ":\"Error finding the requested resource.\" }");
         } catch (IOException | InterruptedException e) {
+            eventProbe.log(INFO, e).counterMetric(POST_ANSWER, 0d);
```

Comment:
Just this should be `eventProbe.log(ERROR, e` since it is a 500 error

Reply:
Added `ERROR` for 5xx errors and for 4xx errors logging set at `INFO` level